packages/fields-cloudinary-image/src/test-fixtures.js

```diff
@@ -1,6 +1,247 @@
+          absolute: true,
+        });
+        const image = prepareFile(filePath[0]);
+        const data = await createItem({
+          keystone,
+          listKey,
+          item: { name: 'Keystone loves React', image },
+          returnFields: 'id image { originalFilename }',
+        });
+        expect(data).not.toBe(null);
+        expect(data.image.originalFilename).toEqual('react.jpeg');
+      })
+    )
+  );
+
+  test(
+    'Read',
+    withKeystone(
+      withHelpers(async ({ keystone, listKey, items }) => {
+        const data = await getItem({
+          keystone,
+          listKey,
+          itemId: items[0].id,
+          returnFields: 'id image { originalFilename }',
+        });
+
+        expect(data).not.toBe(null);
+        expect(data.image.originalFilename).toBe(items[0].image.originalFilename);
+      })
+    )
+  );
+
+  describe('Update', () => {
+    test(
+      'Updating the value',
+      withKeystone(
+        withHelpers(async ({ keystone, items, listKey }) => {
+          const filePath = globby.sync('packages/**/src/**/File/test-files/keystone.jpeg', {
+            absolute: true,
+          });
+          const image = prepareFile(filePath[0]);
+          const data = await updateItem({
+            keystone,
+            listKey,
+            item: {
+              id: items[0].id,
+              data: { image },
+            },
+            returnFields: 'image { originalFilename }',
+          });
+          expect(data).not.toBe(null);
+          expect(data.image.originalFilename).toBe('keystone.jpeg');
+        })
+      )
+    );
+
+    test(
+      'Updating the value to null',
+      withKeystone(
+        withHelpers(async ({ keystone, items, listKey }) => {
+          const data = await updateItem({
+            keystone,
+            listKey,
+            item: {
+              id: items[3].id,
+              data: { image: null },
+            },
+            returnFields: 'image',
+          });
+          expect(data).not.toBe(null);
+          expect(data.image).toBe(null);
+        })
+      )
+    );
+
+    test(
+      'Updating without this field',
+      withKeystone(
+        withHelpers(async ({ keystone, items, listKey }) => {
+          const data = await updateItem({
+            keystone,
+            listKey,
+            item: {
+              id: items[0].id,
+              data: { name: 'Keystone User Guide' },
+            },
+            returnFields: 'name image { originalFilename }',
+          });
+          expect(data).not.toBe(null);
+          expect(data.name).toBe('Keystone User Guide');
+          expect(data.image.originalFilename).toBe(items[0].image.originalFilename);
+        })
+      )
+    );
+  });
+};
+
+// This function will run after all the tests are completed.
+// We use it to cleanup the resources (e.g Cloudinary images) which are no longer required.
+export const afterAll = () => cloudinary.v2.api.delete_resources_by_prefix('cloudinary-test');
```

Comment:
Let's move this code up to the top section of the file. We want to keep all the module config in a single location.